bucket/zeppelin.json

```diff
@@ -17,10 +17,11 @@
         "re": "zeppelin-([\\d.]+)-bin-all.tgz"
     },
     "autoupdate": {
-        "url": "https://archive.apache.org/dist/zeppelin/zeppelin-$version/zeppelin-$version-bin-all.tgz#/zeppelin.tar.gz",
+        "url": "https://www.apache.org/dist/zeppelin/zeppelin-$version/zeppelin-$version-bin-all.tgz#/zeppelin.tar.gz",
         "extract_dir": "zeppelin-$version-bin-all",
         "hash": {
-            "url": "$url.sha512"
+            "url": "$url.sha512",
+            "find": "^($sha512)"
```

Comment:
`"find": "^($sha512)"` is not required, scoop knows the file format and extracts the checksum automatically.